src/epcpm/cli/sunspectostaticmodbus.py

```diff
@@ -136,15 +154,20 @@ def cli(input_sunspec_filename, output_staticmodbus_filename):
             input_sunspec_csv
         )
 
+        # Create root object to store the static modbus model.
+        root = epcpm.staticmodbusmodel.Root()
+
         # Transform input data from sunspec to staticmodbus output.
         for data_row in input_sunspec_csv:
-            root_child = create_function_data(
+            root_child = create_data_object(
                 data_row, scale_factor_uuid_map, sunspec_types, staticmodbus_types
             )
-            data["children"].append(root_child)
+            root.children.append(root_child)
+
+        # Output staticmodbus JSON file using graham.
+        with open(output_staticmodbus_filename, "w", newline="\n") as f:
+            staticmodbus_out = graham.dumps(root, indent=4).data
```

Comment:
comment for what (indent=4) means

Reply:
Added comment in 7b4d874c64ffb3e7af342ef73218e4c7a491d14b.